ueber-uns.html

```diff
@@ -0,0 +1,158 @@
+---
+layout: default
+title: Ueber uns
+---
+
+<section class="primary claim container">
+  <h1 class="display-6">
+    <strong>Wir bringen Nachwuchstalente zur Spitzenleistung!</strong>
+    <br />
+    Wir sind davon überzeugt, dass Programmieren nicht nur eine Fähigkeit,
+    sondern ein
+    <strong>Schlüssel zur Verwirklichung von Träumen</strong> ist.
+  </h1>
+</section>
+<div class="story container">
+  <div class="card overflow-hidden">
+    <div class="row g-0">
+      <div class="image col-md-6 d-flex flex-column justify-content-end">
+        {% picture about-me history.png --alt unsere Geschichte %}
+      </div>
+      <div class="col-md-6 story-position">
+        <div class="card-body p-4 px-lg-5">
+          <h2>Unsere Geschichte</h2>
+          <p class="message">
+            Die DevCraft Academy entstand aus der Idee, Menschen, die das
+            Programmieren lieben, und Unternehmen, die nach talentierten
+            Junior-Programmierern suchen, zu verbinden.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<section class="mission-section">
+  <div class="mission container">
+    <div class="row g-0">
+      <div class="col-md-6">
+        <div class="card-body p-4 px-lg-5">
+          <h2>Mission Statement</h2>
+          <p class="message">
+            Wir wollen Leute, die bereits auf Junior-Niveau programmieren
+            können, auf das nächste Level bringen. Unser Fokus liegt darauf,
+            Menschen zu befähigen, ihre Träume und beruflichen Ziele im Bereich
+            der Programmierung zu erreichen.
+          </p>
+        </div>
+      </div>
+      <div class="image col-md-6 d-flex flex-column justify-content-end">
+        {% picture about-me mission.png --alt Mission %}
+      </div>
+    </div>
+  </div>
+</section>
+<section class="values container">
+  <div class="card overflow-hidden">
+    <div class="row g-0">
+      <div class="image col-md-6 d-flex flex-column justify-content-end">
+        {% picture about-me values.png --alt unsere Werte %}

```

Comment:
Du verwendest bei den `picture`s immer `about-me`. Bist du sicher, dass diese Settings für all die Bilder passen? Die Settings dazu findest du in `picture.yaml`. Dort kannst du gerne auch weitere Settings hinzufügen, wenn du denkst, dass es notwendig ist.

Für Grundlagen zu Responsive Images lies dir am besten mal diesen Artikel durch: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images